Makefile

```diff
@@ -35,10 +35,12 @@ gobuild:
 static:
 	./scripts/build
 
+.dockerbuild:
```

Comment:
Is the intent to cache the builder image with this target? I think it would be better named as `builder-image` or something similar as a dependent target name if that's the case.